core/src/main/java/com/datastax/oss/driver/internal/core/config/map/DefaultOptionsMap.java

```diff
@@ -0,0 +1,189 @@
+    source.put(TypedDriverOption.METRICS_SESSION_CQL_REQUESTS_INTERVAL, Duration.ofMinutes(5));
+    source.put(TypedDriverOption.METRICS_SESSION_THROTTLING_HIGHEST, Duration.ofSeconds(3));
+    source.put(TypedDriverOption.METRICS_SESSION_THROTTLING_DIGITS, 3);
+    source.put(TypedDriverOption.METRICS_SESSION_THROTTLING_INTERVAL, Duration.ofMinutes(5));
+    source.put(
+        TypedDriverOption.CONTINUOUS_PAGING_METRICS_SESSION_CQL_REQUESTS_HIGHEST,
+        Duration.ofSeconds(3));
+    source.put(TypedDriverOption.CONTINUOUS_PAGING_METRICS_SESSION_CQL_REQUESTS_DIGITS, 3);
+    source.put(
+        TypedDriverOption.CONTINUOUS_PAGING_METRICS_SESSION_CQL_REQUESTS_INTERVAL,
+        Duration.ofMinutes(5));
+    source.put(TypedDriverOption.METRICS_NODE_ENABLED, Collections.emptyList());
+    source.put(TypedDriverOption.METRICS_NODE_CQL_MESSAGES_HIGHEST, Duration.ofSeconds(3));
+    source.put(TypedDriverOption.METRICS_NODE_CQL_MESSAGES_DIGITS, 3);
+    source.put(TypedDriverOption.METRICS_NODE_CQL_MESSAGES_INTERVAL, Duration.ofMinutes(5));
+    source.put(TypedDriverOption.SOCKET_TCP_NODELAY, true);
+    source.put(TypedDriverOption.HEARTBEAT_INTERVAL, Duration.ofSeconds(30));
+    source.put(TypedDriverOption.HEARTBEAT_TIMEOUT, Duration.ofMillis(500));
+    source.put(TypedDriverOption.METADATA_TOPOLOGY_WINDOW, Duration.ofSeconds(1));
+    source.put(TypedDriverOption.METADATA_TOPOLOGY_MAX_EVENTS, 20);
+    source.put(TypedDriverOption.METADATA_SCHEMA_ENABLED, true);
+    source.put(TypedDriverOption.METADATA_SCHEMA_REQUEST_TIMEOUT, Duration.ofSeconds(2));
+    source.put(TypedDriverOption.METADATA_SCHEMA_REQUEST_PAGE_SIZE, 5000);
+    source.put(TypedDriverOption.METADATA_SCHEMA_WINDOW, Duration.ofSeconds(1));
+    source.put(TypedDriverOption.METADATA_SCHEMA_MAX_EVENTS, 20);
+    source.put(TypedDriverOption.METADATA_TOKEN_MAP_ENABLED, true);
+    source.put(TypedDriverOption.CONTROL_CONNECTION_TIMEOUT, Duration.ofMillis(500));
+    source.put(TypedDriverOption.CONTROL_CONNECTION_AGREEMENT_INTERVAL, Duration.ofMillis(200));
+    source.put(TypedDriverOption.CONTROL_CONNECTION_AGREEMENT_TIMEOUT, Duration.ofSeconds(10));
+    source.put(TypedDriverOption.CONTROL_CONNECTION_AGREEMENT_WARN, true);
+    source.put(TypedDriverOption.PREPARE_ON_ALL_NODES, true);
+    source.put(TypedDriverOption.REPREPARE_ENABLED, true);
+    source.put(TypedDriverOption.REPREPARE_CHECK_SYSTEM_TABLE, false);
+    source.put(TypedDriverOption.REPREPARE_MAX_STATEMENTS, 0);
+    source.put(TypedDriverOption.REPREPARE_MAX_PARALLELISM, 100);
+    source.put(TypedDriverOption.REPREPARE_TIMEOUT, Duration.ofMillis(500));
+    source.put(TypedDriverOption.NETTY_DAEMON, false);
+    source.put(TypedDriverOption.NETTY_IO_SIZE, 0);
+    source.put(TypedDriverOption.NETTY_IO_SHUTDOWN_QUIET_PERIOD, 2);
+    source.put(TypedDriverOption.NETTY_IO_SHUTDOWN_TIMEOUT, 15);
+    source.put(TypedDriverOption.NETTY_IO_SHUTDOWN_UNIT, "SECONDS");
```

Comment:
nit, maybe use a static constant for repeatable values like for example `SECONDS`